tasks.py

```diff
@@ -225,6 +225,12 @@ def implemented_tests_table(c):
     headers = ['Category', 'Test Name', 'Images']
     rows = []
 
+    # Exclude certain tests from the table
```

Comment:
The idea is to exclude all tests from test_infrastructure.py. Is it possible to not include any from there? Then we won't have to remember these.